scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
```

Comment:
:stop_sign: **Security Vulnerability**: Command injection vulnerability detected. This allows arbitrary command execution and is a critical security risk[^2].
```suggestion
// Use path.join for safe file path construction
const safeUserInput = path.normalize(userInput).replace(/^(\.\.(\/|\\|$))+/, '')
const safePath = path.join('/', safeUserInput)
child_process.execFile('ls', [safePath], (error, stdout, stderr) => {
    if (error) {
        console.error(`Error: ${error}`)
        return
    }
    console.log(stdout)
})
```